pkg/cmd/kafka/create/data.go

```diff
@@ -0,0 +1,160 @@
+package create
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/shared/accountmgmtutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
+)
+
+// Types we use on backend to map AMS Quotas
+type CloudProviderId = string
+
+// Additional types that are used in backend to match
+// AMS Quota instance types
+const (
+	// Matches QuotaTrialType = trial
+	DeveloperType CloudProviderId = "developer"
+
+	// Matches QuotaStandardType = standard
+	StandardType CloudProviderId = "standard"
+
+	// Deprecated by DeveloperType
+	TrialType CloudProviderId = "eval"
+)
+
+// mapAmsTypeToBackendType - Cloud providers API is not using AMS types but some other values (CloudProviderValues)
+func mapAmsTypeToBackendType(amsType *accountmgmtutil.QuotaSpec) CloudProviderId {
```

Comment:
Area of interest 1

Reply:
Do we plan to update Account Management Service types to match the kas-fleet-manager types so we can avoid this mapping on the client side? If not, we probably need to add the Account Management Service type to the kas-fleet-manager type. Then the client would need to create a map still, but wouldn't have to hardcode all the mapped values.

Reply:
Agree. That will reduce business logic on the client by half. 

Additionally AMS should probably not return us instances that we cannot create (like Trial instance when standard is there), however I'm not sure if that would be possible to implement on AMS side